.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
   - KONG_VERSION=0.10
   - KONG_VERSION=0.11
   - KONG_VERSION=0.12
+  - KONG_VERSION=0.13.0
```

Comment:
Why not use "0.13" instead of 0.13.0?

Reply:
0.13 should be used. It makes sure tests are always run against the last patched version.